bin/opm

```diff
@@ -2195,6 +2198,57 @@ sub rebase_path ($$$) {
     return undef;
 }
 
+sub do_clean {
+    my @valid_options = ('dist');
```

Comment:
We won't have many different kinds of arguments, especially when we only have one right now. No need to introduce a Perl array here. It is just referenced once and simply adds line noises IMHO :)